tools/env_probe_vc_inv.rb

```diff
@@ -1,85 +0,0 @@
-require 'VMwareWebService/MiqVim'
-require 'dbi'
```

Comment:
Fails to load:
/Users/joerafaniello/.gem/ruby/2.4.4/gems/activesupport-5.0.7/lib/active_support/dependencies.rb:293:in `require': cannot load such file -- dbi (LoadError)